.github/workflows/docker-setup/run-ee-server.bash

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Prevent Windows shell from mangling paths
+export MSYS_NO_PATHCONV=1
+
+# Input defaults
+
+BASE_IMAGE=${BASE_IMAGE:-"aerospike/aerospike-server-enterprise"}
+# Server features
+MUTUAL_TLS=${MUTUAL_TLS:-"0"}
+SECURITY=${SECURITY:-"0"}
+STRONG_CONSISTENCY=${STRONG_CONSISTENCY:-"0"}
+
+# End inputs
+
+VOLUME_NAME=aerospike-conf-vol
+docker volume create $VOLUME_NAME
+
+volume_dest_folder="/workdir"
+container_name_for_populating_volume="container_for_populating_volume"
+
+docker run --name $container_name_for_populating_volume --rm -v $VOLUME_NAME:$volume_dest_folder -d alpine tail -f /dev/null
+docker cp ./ $container_name_for_populating_volume:$volume_dest_folder
+docker stop $container_name_for_populating_volume
+
+aerospike_yaml_file_name="aerospike-dev.yaml"
+
+call_from_yq_container() {
+    # alpine container's process is run as root user
+    # So the files copied into the named volume will also be owned by root
+    # Since these files only have write permission for the owner (root),
+    # We also need to run yq container as root in order to write to the yaml file in this volume.
+    docker run --rm --user root -v $VOLUME_NAME:$volume_dest_folder mikefarah/yq "$1" -i ${volume_dest_folder}/${aerospike_yaml_file_name}
+}
+
```

Comment:
commented in wait-for-as-server-to-start but there are multiple patterns doing the same thing but in different ways. I suggest to find the one that works and be consistent.